UPGRADE.md

```diff
@@ -1,5 +1,22 @@
 # Upgrade
 
+## dev-develop
+
+### Texteditor
+
+The params for the texteditor content type where changed.
+
+| Before                                        | Now                                                                                                         |
+|-----------------------------------------------|-------------------------------------------------------------------------------------------------------------|
+| `<param name="tables" value="true" />`        | `<param name="table" value="true" />`                                                                       |
+| `<param name="links" value="true" />`         | `<param name="link" value="true" />`                                                                        |
+| `<param name="pasteFromWord" value="true" />` | `<param name="paste_from_word" value="true" />`                                                             |
+| `<param name="maxHeight" value="500" />`      | `<param name="max_height" value="500" />`                                                                   |
+|                                               |                                                                                                             |
+| `<param name="iframes" value="true" />`       | iframe and script tags can activated with an ckeditor parameter:                                             |
```

Comment:
@alexander-schranz "iframe and script tags can activated with an ckeditor" ... can you explain it in more detail